app/controllers/animal_sightings_controller.rb

```diff
@@ -36,6 +38,9 @@ def update
 
     private
     def sight_params
-        params.require(:animal_sighting).permit(:latitude, :longitude, :date,:animal_id)
+        params.require(:animal_sighting).permit(:latitude, :longitude, :date, :animal_id)
+    end
+    def sightings_params
+        params.require(:animal_sightings).permit(:start_date, :end_date)
```

Comment:
I don't see this being called anywhere, do you need this?

Reply:
Commited new file with changes